zenodo/modules/deposit/templates/zenodo_deposit/edit.html

```diff
@@ -140,6 +140,35 @@ <h1>Edit upload</h1>
       <h1>New version</h1>
       {%- else %}
       <h1>New upload</h1>
+      {% if record.files %}
+      {%- if current_user_is_admin() %}
+      <div class="btn-group btn-block">
+        <button type="button" class="btn btn-m btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Admin <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu">
+          <li><a href="{{ url_for('persistentidentifier.index_view', flt1_21=record.id) }}">
+            Persistent identifiers
+          </a></li>
+          <li><a href="{{ url_for('recordmetadata.details_view', id=record.id) }}">
+            Record
+          </a></li>
+          <li><a href="{{ url_for('bucket.edit_view', id=record.files.bucket) }}">
+            Bucket
+          </a></li>
+          {%- endif %}
```

Comment:
Hm, I can't find where the matching `if ` of this block is. Can you double-check?